### PR TITLE
pop screen

### PR DESCRIPTION
**Pull request by Kiran2821996**

### CODE REVIEW

File: src/Components/trending/TrendItems.jsx [hunk 1]
```diff
@@ -52,7 +28,10 @@ function TrendItems({ data }) {
           <div
             className="trend_block_item"
             key={idx}
-            onClick={(e) => handleClick(item.item.id)}
+            onClick={(e) => {
+              handleClick(item.item.id);
+              setModal2Open(true);
```
Comment (Souvikdeb2612) [inline src/Components/trending/TrendItems.jsx]: create a function and call here

File: src/components/trending/Pop.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  objectData.sort((a, b) => {
+    return a.day - b.day;
+  });
+
+  return (
+    <Modal
+      centered
+      visible={modal2Open}
+      footer={null}
+      onCancel={() => setModal2Open(false)}
```
Comment (Souvikdeb2612) [inline src/components/trending/Pop.jsx]: create a function and call here

File: src/components/trending/Pop.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    >
+      {data?.map((item) => {
```
Comment (Souvikdeb2612) [inline src/components/trending/Pop.jsx:42]: destructure item here

File: src/components/trending/TrendItems.jsx [hunk 4]
```diff
@@ -52,7 +28,10 @@ function TrendItems({ data }) {
           <div
             className="trend_block_item"
             key={idx}
-            onClick={(e) => handleClick(item.item.id)}
+            onClick={(e) => {
+              handleClick(item.item.id);
+              setModal2Open(true);
```
Comment (Souvikdeb2612) [inline src/components/trending/TrendItems.jsx]: same as above

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/trending/TrendItems.jsx (hunk 1 above)
Comment (Kiran2821996) [reply src/Components/trending/TrendItems.jsx]: done

File: src/components/trending/Pop.jsx (hunk 2 above)
Comment (Kiran2821996) [reply src/components/trending/Pop.jsx]: done

File: src/components/trending/TrendItems.jsx (hunk 4 above)
Comment (Kiran2821996) [reply src/components/trending/TrendItems.jsx]: done

File: src/Components/trending/TrendItems.jsx [hunk 5]
```diff
@@ -1,48 +1,25 @@
...
 
+import Pop from "./Pop";
+
 function TrendItems({ data }) {
   const [dataShow, setDataShow] = useState([]);
-  const [dataClick, setdataClick] = useState("BTC");
-  const [toggle, setToggle] = useState(false);
+  const [dataClick, setdataClick] = useState();
+  const [interval, setInterval] = useState(10);
+  const [modal2Open, setModal2Open] = useState(false);
```
Comment (Souvikdeb2612) [inline src/Components/trending/TrendItems.jsx:10]: State name isn't meaningful

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**